docs/source/configuration/advanced_configuration.md

```diff
@@ -285,3 +290,58 @@ CONFIG_LOADER_ARGS = {
 
 If no merge strategy is defined, the default destructive strategy will be applied. Note that this merge strategy setting only applies to configuration files in **different** environments.
 When files are part of the same environment, they are always merged in a soft way. An error is thrown when files in the same environment contain the same top-level keys.
+
+
+## Advance Configurations without Kedro Project
+By defaults, Kedro Project has a `base` and `local` environments. In some cases, you may only want to use the `OmegaConfigLoader` without a Kedro project. This is possible because Kedro embrace [modularity at the core](https://github.com/kedro-org/kedro/wiki/Kedro-Principles#1-modularity-at-the-core-%EF%B8%8F).
```

Comment:
```suggestion
By default, a Kedro project has a `base` and `local` environment. In some cases, you may only want to use the `OmegaConfigLoader` without a Kedro project. This is possible because Kedro embraces [modularity at the core](https://github.com/kedro-org/kedro/wiki/Kedro-Principles#1-modularity-at-the-core-%EF%B8%8F).
```

Reply:
I think this probably needs a bit more attention. First, I'd suggest we reorder the argument so it's presented as:

* In some cases you may want to use the config loader without a full Kedro project (with a link to docs about this or further explanation of the use cases that mean you'd want to do it and where/how to do it)
* But by default there's an expectation of `base` and `local`

I'd remove the sentence about modularity.